happypanda/gallerydb.py

```diff
@@ -27,15 +27,15 @@
 
 from PyQt5.QtCore import QObject, pyqtSignal, QTime
 
-from utils import (today, ArchiveFile, generate_img_hash, delete_path,
+from happypanda.utils import (today, ArchiveFile, generate_img_hash, delete_path,
```

Comment:
Thing is, I can either fix the indent or fix it the proper way (No single-line multiple imports)

Doing it the first way would make it match the rest and be PEP8 compliant, but doing it the second way would be PEP8 compliant and not look ugly.

Thing is it would then